tests/testthat/test-estimate_KM.R

```diff
@@ -1,31 +1,29 @@
-#' @title Specifications estimate_KM
-#' @section Last updated by:
-#' Steven Haesendonckx
-#' @section Last update date:
-#' 12-OCT-2021
-
-# Specifications ----------------------------------------------------------
-
+#' @title Specifications test-estimate_KM.R
+#' @section Last updated by: Tim Treis (tim.treis(at)outlook.de)
+#' @section Last update date: 2021-10-28 16:29:24
+#'
+#' @section List of tested specifications
 #' T1. The function accepts a `data.frame` `tibble` or `data.table`
-#' T1.1 No error when `data` is a data.frame
-#' T1.2 No error when `data` is a tibble
-#' T1.3 No error when `data` is a data.table
-#' T1.4 An error when `data` is a random object
+#' T1.1. No error when `data` is of class `data.frame`
+#' T1.2. No error when `data` is of class `tibble`
+#' T1.3. No error when `data` is of class `data.table`
+#' T1.4. An error when `data` is of class `list`
```

Comment:
This is indeed better, but I would keep it general eg unexpected class eg 'list'